spec/requests/admin/configs_spec.rb

```diff
@@ -807,15 +807,17 @@
         end
 
         it "unsets appropriate SMTP config, and apply default value if applicable" do
-          Settings::SMTP.address = "smtp.example.com"
-          Settings::SMTP.port = 12_345
+          default_address = ApplicationConfig["SMTP_ADDRESS"].presence || nil
```

Comment:
Hey MK, locally and on Travis, this spec kept failing, specifically the expectation that the default port was `25` (locally it was `2525` for me). I thought it more robust to expect whatever the default port was on the system running the spec. Same for the default address.

Reply:
😄 I understand and agree with the reasoning. I was more wondering why we use `ApplicationConfig` directly instead of

```
Settings::SMTP.get_field(:smtp_port)[:default]
```

Which is a bit more general.

Reply:
I will change it then @citizen428 😅  I understand the point you are making now.

Reply:
Hey @citizen428,

So locally, this is what I get for each of these Settings values:

```
Settings::SMTP.get_field(:smtp_address)[:default]
=> nil

Settings::SMTP.get_field(:smtp_port)[:default]
=> nil

ApplicationConfig["SMTP_PORT"]
=> "2525"

ApplicationConfig["SMTP_ADDRESS"]
=> "smtp.mailtrap.io"
```

The expectations in the relevant `configs_spec.rb` specs are now failing when I use the more general `Settings::SMTP.get_field`.

I'm not sure why these values differ from each other. I think I'll keep use of `ApplicationConfig`; thoughts?

Reply:
It's fine to use `ApplicationConfig` of course 😃